src/main/java/com/kodilla/ecommercee/domain/ProductGroup.java

```diff
@@ -29,9 +27,12 @@
     @OneToMany(targetEntity =  Product.class,
             mappedBy = "productGroup",
             cascade = CascadeType.ALL,
-            fetch = FetchType.LAZY)
+            fetch = FetchType.EAGER)
```

Comment:
Dlaczego zmiana na EAGER?

Reply:
w jednym z poprzednich tasków chyba dotyczących testów @pszczepcio dostał taką informację żeby wszędzie zmienić 
fetchType na EAGER ponieważ testy nie działały prawidłowo

Reply:
zgadza się, ale to jest task którym zajmuje się tylko jedna osoba, po z mergowaniu każdy będzie miał te zmiany u siebie. Nie powinno się mieszać tasków ze sobą tak jak jest to tutaj bo doprowadza to do konfliktów

Reply:
ok, po prostu zasugerowałem się czymś innym zamiast robić swoje